features/util.py

```diff
@@ -325,3 +325,13 @@ def emit_spinner():
     finally:
         print()
         dot_process.terminate()
+
+
+class SafeLoaderWithoutDatetime(yaml.SafeLoader):
+    pass
+
+
+SafeLoaderWithoutDatetime.yaml_implicit_resolvers = {
```

Comment:
can't we define this as a class property ?

Reply:
Done!